internal/indexmanagement/scripts.go

```diff
@@ -207,18 +256,8 @@ fi
 
 echo "Checking if $nextIndex is the write index for ${POLICY_MAPPING}-write"
 
-# if true, ensure write-alias points to next index
-writeIndices=$(curl -s $ES_SERVICE/${POLICY_MAPPING}-*/_alias/${POLICY_MAPPING}-write \
-  --cacert /etc/indexmanagement/keys/admin-ca \
-  --connect-timeout ${CONNECT_TIMEOUT} \
-  -H"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-  -HContent-Type:application/json \
-  --retry 5 \
-  --retry-delay 5)
-
-jsonResponse="$(echo [$writeIndices] | sed 's/}{/},{/g')"
-
-if ! writeIndex="$(python /tmp/scripts/getWriteIndex.py "${POLICY_MAPPING}-write" "$jsonResponse")" ; then
+## if true, ensure write-alias points to next index
+if ! writeIndex="$(getWriteIndex "${POLICY_MAPPING}-write")" ; then
```

Comment:
Can we spell out the `echo $writeIndex` as a proper exit message to understand what failed where?